test/unit/portal-target.spec.js

```diff
@@ -144,7 +152,8 @@ describe('PortalTarget', function () {
     })
     return vm.$nextTick().then(() => {
       td.verify(spy({
-        to: 'target', from: 'source', passengers: vNodes },
+          to: 'target', from: 'source', passengers: vNodes
```

Comment:
this should now receive arrays. `spy([ ... ], [ ... ])`.

This also has to be changed in the `emitChange` method in `PortalTarget`.